app/controllers/public_controller.rb

```diff
@@ -19,7 +19,7 @@ def profile
     @profile = Account.find_by_username params[:username]
     @posts = @profile.posts
     @my_comments = Comment.where(account_id: @profile.id)
-    @my_posts = Post.where(account_id: @profile.id).page(params[:page]).per 5
+    @my_posts = Post.where(account_id: @profile.id).where(is_drafted: false).page(params[:page]).per 5
```

Comment:
you don't need 2 where, can you remove one?

Reply:
Removed @iMagesh